README.md

```diff
@@ -48,6 +48,12 @@ Plugin requires following events in GitLab webhook:
 - Merge Request Hook
 - Note Hook
```

Comment:
Travis config should be added.
Then add travis badge (to https://travis-ci.org/node-ci/nci-gitlab-webhook) to the end of the first readme section.